themes/root/templates/icons/FontAwesome.phtml

```diff
@@ -0,0 +1 @@
+<i class="fa fa-<?=$this->icon ?>"<?=$this->attrs ?> aria-hidden="true"></i>
```

Comment:
We should probably escapeHtmlAttr() the icon value here (and also in Remix.phtml) just to be on the safe side.

Reply:
Is it enough to escape them in the helper? https://github.com/vufind-org/vufind/pull/1926/files#diff-b10064b350845d825e6677a96d797f7c11bc5950d0b7ffe061176b4d2e440d14R83-R86

Reply:
Yes, the attributes are fine... it's $this->icon that doesn't appear to be getting escaped anywhere. (Admittedly, it shouldn't NEED escaping, but it doesn't hurt in case of weird edge cases or misconfigurations).